front/app/containers/MainHeader/Components/NavbarContent/DesktopNavbarContent/index.tsx

```diff
@@ -0,0 +1,273 @@
+import React from 'react';
+
```

Comment:
I moved all this code into a new component so I could separate the desktop and mobile nav bars, but I haven't changed any of the original code in this file.